src/services/bulkstatus.service.js

```diff
@@ -91,7 +91,7 @@ async function checkBulkStatus(req, res) {
               display: 'New resource created'
             }
           ],
-          text: 'Bulk import successfully completed'
+          text: `Bulk import successfully completed, successfully imported ${bulkStatus.exportedResourceCount}`
```

Comment:
Technically, `exportedResourceCount` is not always guaranteed to be here. You can see in the code that we have a few checks where it can be `-1`.

This value is computed by looking at the `count` property returned by the initial `$export` request as part of PnP. The export server does not have to specify this count, though they can if they want to. So this code will break if we are dealing with an export server that doesn't include `count` on the exported ndjson, and it will say successfully imported -1.

What we should be doing is grabbing the count of resources actually inserted by the ndjson worker job and tracking that information. We can check the number of promises resolved by inspecting how many results came back from `Promise.all`. Then we can use that to update the bulkStatus information in the db. We shouldn't be relying on `bulkStatus.exportedResourceCount`

For future work with the error handling stuff too, it's important that we get the count in the ndjson worker itself, because we want to make sure that we get a true value on how many mongo insertions were successfully done. What this current code is doing is saying how many resources we _expected_ to have inserted, not actually inserted